pkg/cmd/cli/cmd/rsync/copyrsync.go

```diff
@@ -34,16 +34,15 @@ var rshExcludeFlags = sets.NewString("delete", "strategy", "quiet", "include", "
 
 func newRsyncStrategy(f *clientcmd.Factory, c *cobra.Command, o *RsyncOptions) (copyStrategy, error) {
 	// Determine the rsh command to pass to the local rsync command
-	rsh := cmdutil.SiblingCommand(c, "rsh")
-	rshCmd := []string{rsh}
+	rshCmd := cmdutil.SiblingCommand(c, "rsh")
```

Comment:
SiblingCommand returns a []string that can be iterated over when passed to escapeRsyncCommand.  Am I just missing something here?